src/main/webapp/app/exam/manage/exams/exam-mode-picker/exam-mode-picker.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Exam } from 'app/entities/exam.model';
+
+@Component({
+    selector: 'jhi-exam-mode-picker',
+    templateUrl: './exam-mode-picker.component.html',
+    styleUrls: ['./exam-mode-picker.component.scss'],
+})
+export class ExamModePickerComponent {
+    @Input() exam: Exam;
+    @Input() disableInput: boolean;
+
+    @Output() ngModelChange = new EventEmitter();
```

Comment:
This name is sufficient in general, but a bit abstract imo. Consider changing it to something like e.g. `examModeChanged` as it would make things easier to understand for developers not familiar with this component.